### ISSUE TITLE
kube-prometheus v0.1.0’s manifests install v0.29.0 of the operator, not v0.30.0

### ISSUE THREAD

**Issue by MerelyAPseudonym** [coreos/kube-prometheus#105]
**What happened?**

Despite the the intent [in #101] to upgrade kube-prometheus to use prometheus-operator 0.30.0, the contents of `manifests/` result in version 0.29.0 being installed instead.

[in #101]: https://github.com/coreos/kube-prometheus/pull/101/commits/3ecedc64511794ecca51c601235dcb846e843023

**Did you expect to see some different?**

Yes, after following the instructions in https://github.com/coreos/kube-prometheus/blob/v0.1.0/README.md#quickstart, I expected version 0.30.0 of the Prometheus Operator to be deployed to my Kubernetes cluster.

**How to reproduce it (as minimally and precisely as possible)**:
Rather than reproduce _installation_ instructions, I'll just include evidence that the manifests are incompletely updated:
```sh-session
$ cd kube-prometheus/
$ git checkout v0.1.0
Note: checking out 'v0.1.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at a536114 Merge pull request #101 from metalmatze/cut-v0.1
$ grep 'image: quay.io/coreos/prometheus-operator' manifests/0prometheus-operator-deployment.yaml
        image: quay.io/coreos/prometheus-operator:v0.29.0
```

**Anything else we need to know?**:
I'll have one or two PRs for submitted momentarily with _proposed_ fixes.
(I omitted the environment section from the GitHub issue template since it wasn’t relevant to this particular bug.)

**Comment by BenoitKnecht** [comment]
This was fixed in #106, merged in branch `release-0.1`. There hasn't been a tagged release on that branch since v0.1.0, but I guess users who need support for k8s < 1.14 should follow the `release-0.1` branch rather than stay on a specific tag.

**Comment by brancz** [comment]
Agreed @BenoitKnecht . Thanks!
